web-ts/docs/technical-documentation.md

```diff
@@ -0,0 +1,94 @@
+# Technical Documentation
+
+## Contents
```

Comment:
This is copied from web project. Deleted a little bit of flow stuff and added the Folder-Structur Part. I think we should now think carefully about how we want to have it. This is my suggestion.